src/Commands/AzureBestPractices/AzureBestPracticesGetCommand.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using AzureMcp.Arguments;
+using AzureMcp.Models.Command;
+using Microsoft.Extensions.Logging;
+using ModelContextProtocol.Server;
+
+
+namespace AzureMcp.Commands.BestPractices;
+
+public sealed class AzureBestPracticesGetCommand(ILogger<AzureBestPracticesGetCommand> logger) : BaseCommand()
+{
+    private readonly ILogger<AzureBestPracticesGetCommand> _logger = logger;
+
+    private static readonly string _bestPracticesText = LoadBestPracticesText();
+
+    private static string GetBestPracticesText() => _bestPracticesText;
+
+    private static string LoadBestPracticesText()
+    {
+        var assembly = typeof(AzureBestPracticesGetCommand).Assembly;
+        const string resourceName = "AzureMcp.Resources.azure-best-practices.txt";
+
+        using var stream = assembly.GetManifestResourceStream(resourceName)
+            ?? throw new InvalidOperationException($"Unable to load embedded resource: {resourceName}");
+
+        using var reader = new StreamReader(stream);
+        return reader.ReadToEnd();
+    }
```

Comment:
Please add a helper to the project so we can just pass the file name and it returns the string.

Reply:
added it.